### PR TITLE
Create OWNERS for Kubeflow Getting Started Workstation

### PR DESCRIPTION
**Pull request by 8bitmp3**
This PR adds a new separate OWNERS file with approvers and reviewers for Workstation to assist with Issues/PRs in that directory. It's a copy of Getting Started's OWNERS with an addition of @alfsuse .

/assign @RFMVasconcelos

### CODE REVIEW

**Comment by RFMVasconcelos** [comment]
Hey @8bitmp3! 

Thank you for the push on this front! 

/lgtm
/approve

File: content/en/docs/started/workstation/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+approvers:
+  - RFMVasconcelos
+  - 8bitmp3
+
+reviewers:
+  - alfsuse
```
Comment (Bobgy) [inline content/en/docs/started/workstation/OWNERS:6]: Note, OWNERS inherit through the tree, you don't need to duplicate anyone here

Comment (Bobgy) [reply content/en/docs/started/workstation/OWNERS:6]: @8bitmp3

Comment (8bitmp3) [reply content/en/docs/started/workstation/OWNERS:6]: Thanks @Bobgy !

**Pull request merged**